### PR TITLE
mainly MQTT fixes for Domoticz

### PR DESCRIPTION
**Pull request by proddy**

### CODE REVIEW

File: src/devices/thermostat.cpp [hunk 1]
```diff
@@ -4912,7 +4912,7 @@ void Thermostat::register_device_values_dhw(std::shared_ptr<Thermostat::DhwCircu
                               DeviceValueUOM::MINUTES,
                               MAKE_CF_CB(set_wwchargeduration));
         register_device_value(tag, &dhw->wwCharge_, DeviceValueType::BOOL, FL_(wwCharge), DeviceValueUOM::NONE, MAKE_CF_CB(set_wwcharge));
-        register_device_value(tag, &dhw->wwExtra_, DeviceValueType::UINT8, FL_(wwExtra), DeviceValueUOM::DEGREES);
+        register_device_value(tag, &dhw->wwExtra_, DeviceValueType::UINT8, FL_(wwExtra), DeviceValueUOM::NONE);
```
Comment (MichaelDvP) [inline src/devices/thermostat.cpp:4915]: No complain in all these years ;-) 
But the comment in https://github.com/emsesp/EMS-ESP32/blob/9689f0a7da258d5cc27cb9e8a588f98a736a3141/src/devices/thermostat.cpp#L1249 says it is bool.

**Review by MichaelDvP — APPROVED** [review]
Looks all good to me.

File: src/devices/thermostat.cpp (hunk 1 above)
Comment (proddy) [reply src/devices/thermostat.cpp:4915]: that's what I thought, but if you scroll up you see it also can have a value of 2. So?? not sure

**Comment by proddy** [comment]
made some more edits. While we wait for the Domoticz to be tested - https://github.com/emsesp/EMS-ESP32/issues/2177

File: src/devices/thermostat.cpp (hunk 1 above)
Comment (MichaelDvP) [reply src/devices/thermostat.cpp:4915]: Did not find this. The comment above describes the telegram offsets.
I think it's bool.

Comment (proddy) [reply src/devices/thermostat.cpp:4915]: i think you're right. will change

**Pull request merged**